### DISCUSSION TITLE
Getting error with workflow file

### DISCUSSION THREAD

**Discussion by Margen67** [community/community#25816] [Actions]
<p><a href="https://github.com/Margen67/cheat-engine/blob/actions/.github/workflows/main.yml" rel="nofollow noopener">https://github.com/Margen67/cheat-engine/blob/actions/.github/workflows/main.yml</a></p>
<p>This is the error:</p>
<p><a href="https://github.com/Margen67/cheat-engine/runs/231080800" rel="nofollow noopener">https://github.com/Margen67/cheat-engine/runs/231080800</a></p>
<pre><code>- Your workflow file was invalid: Unrecognized named-value: 'G15'. Located at position 19 within expression: matrix['arch'] == G15
</code></pre>
<p>Am I doing something wrong?</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi,</p>
<p>The env is not included in the matrix but involved as ‘matrix.paramter’ for the command, so they are not known. I copied your repro and make the code change as below, the workflows started as expected, thanks.</p>
<pre><code>jobs:
  build:
    runs-on: windows-latest
    strategy:
      matrix:
        arch: [64, 32]
        lazarus-ver: [lazarus-2.0.4-61666]
        fpc-ver: [fpc-3.2.0-beta-42593]
        include:
          - arch: 64
            cpu: x86_64
            os: win64
          - arch: 32
            cpu: i386
            os: win32
</code></pre>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/8/8b54e2fd7abb0a84c7cc38352b8aef6f35d7f212.png" data-download-href="/uploads/short-url/jSAeI3aBFFaXPldT1q5ldcJ02ky.png?dl=1" title="excuted.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/52772761/181093600-5f938019-4623-4baf-8ae2-0536f69df866.png" alt="excuted.png" data-base62-sha1="jSAeI3aBFFaXPldT1q5ldcJ02ky" width="690" height="20" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/8/8b54e2fd7abb0a84c7cc38352b8aef6f35d7f212_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">excuted.png</span><span class="informations">1488×45 3.83 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
